templates/docs/examples/patterns/lists/lists-dividers-dotted.html

```diff
@@ -0,0 +1,16 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Lists / Dotted with horizontal divider{% endblock %}
+
+{% block standalone_css %}patterns_lists{% endblock %}
+
+{% block content %}
+<ul class="p-list--divided">
+  <li class="p-list__item is-dotted">Balance compute load in the cloud</li>
+  <ul class="p-list--divided">
```

Comment:
That is not valid markup.

Only `<li>` can be a child of ul/ol. So the nested list needs to be inside `<li>`.

```
<ul class="p-list--divided">
  <li class="p-list__item is-dotted">Balance compute load in the cloud
     <ul class="p-list--divided">
     </ul>
   </li>
</ul>
 ```

Reply:
That's what I did at first but then I couldn't manage to style it correctly without using the `:has()` pseudo-selector

Reply:
We can't build on top of invalid markup, so we have to figure out the way to do it.

What we can do in the worst case it to have a class name `is-nested` or `has-nested-list` or something like that. But hopefully this would not be needed.